packages/mobile/src/send/SendConfirmation.test.tsx

```diff
@@ -83,20 +84,18 @@ describe('SendConfirmation', () => {
     )
 
     // Initial render.
+    // Await to force fees to render prior to checking snapshot and attempting press
+    await waitForElement(() => fireEvent.press(tree.getByText('feeEstimate')))
+    // TODO (anton): line 744 of the snapshot 'rotate: ...' sometimes differs for unknown reasons
```

Comment:
One line of the snapshot appears to be nondeterministic. Currently the snapshot at this line has `'rotate': 0deg` which appears to be a common result, though it is not guaranteed.